src/components/widgets/EventFilters/EventFilters.tsx

```diff
@@ -161,7 +73,9 @@ export const EventFilters = () => {
                 <CreateTag eventTags={[]} onChange={handleTagsChange} />
             </div>
 
-            {activeTab === "list" && <FilteredEvents data={filteredData?.eventFilters} sizeCard={SizeCard.ML} />}
+            {activeTab === "list" && (
```

Comment:
Вроде так тоже можно:

```suggestion
            {activeTab === NavbarEventTabs.LIST && (
```
тогда будет более читаемым

Reply:
Исправил